spartan/aztec-network/templates/transaction-bot.yaml

```diff
@@ -124,6 +144,10 @@ spec:
 
               export BOT_ACCOUNT_SALT=$(echo $K8S_POD_NAME | awk -F'-' '{print $NF+1}') # +1 so that salt is not 0
 
+              # set this to a folder in order to capture simulation recordings for later debug
+              # be sure to set it to a persistent volume
+              export CIRCUIT_RECORD_DIR=""
```

Comment:
Left this here, in the command section, because we might want to turn it on only for specific bots (e.g. we had trouble with bot-0 last time)